docs/swagger-1.9.json

```diff
@@ -399,6 +407,7 @@
           "user"
         ],
         "summary": "report a user with reason",
```

Comment:
Casing inconsistent? First letter should be capitalized for all summaries. Happens above as well.